custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -103,7 +103,7 @@ def get_data_for_daily_usage(date, filters):
                     'help_text': awcs_launched_help_text(),
                     'color': None,
                     'percent': None,
-                    'value': get_value(awc_this_month_data, 'awcs'),
+                    'value': get_value(daily_yesterday if current_month_selected else awc_this_month_data, 'awcs'),
```

Comment:
cal and i offlined. good to get this in for the demo, but a different fix will need to be applied as this bug was found as a result of the celerybeat timezone issues we found when upgrading to 4.1